.github/workflows/build-and-test-differential.yaml

```diff
@@ -78,11 +78,20 @@ jobs:
         id: get-modified-packages
         uses: autowarefoundation/autoware-github-actions/get-modified-packages@v1
 
+      - name: Get modified files
+        id: get-modified-files
+        uses: tj-actions/changed-files@v28
+        with:
+          files: |
+            **/*.cpp
+            **/*.hpp
+
       - name: Run clang-tidy
-        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+        if: ${{ steps.get-modified-files.outputs.all_changed_files != '' }}
         uses: autowarefoundation/autoware-github-actions/clang-tidy@v1
         with:
           rosdistro: humble
           target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
```

Comment:
If `target-files` is specified, `target-package` will be ignored, so I think `get-modified-packages` step is not necessary.

https://github.com/autowarefoundation/autoware-github-actions/pull/178/files#diff-7cfc87275896c2e69225a6c08de4e7f897ffd6e5f2d32319a7547ed18f90e682R100-R105

Reply:
`target-package` is necessary for setting up the build environment.
![image](https://user-images.githubusercontent.com/31987104/187846753-fcfd2f1b-c6c4-47a8-aba1-c33822479cb7.png)

Reply:
Ah, I see.